.travis.yml

```diff
@@ -10,7 +10,7 @@ os:
 - linux
 
 go:
-  - "1.12.x"
+  - "1.12.13"
```

Comment:
same as https://github.com/containerd/containerd/pull/3620#discussion_r344307012

Changed this to be an explicit version as well; thought that if we'd have to update the AppVeyor config anyway, so might as well pin both travis and appveyor to use an explicit version